### DISCUSSION TITLE
User Agent string reduced In Chrome 100+ <title>

### DISCUSSION THREAD

**Discussion by magicse** [community/community#45970] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After update to Chrome 109 I didn't see reaction icons. only squares. 
[Chrome is deprecating navigator.userAgent ](https://developer.chrome.com/en/blog/user-agent-reduction-deprecation-trial/)
After installing User-Agent switcher plugin for chrome and force setting for user agent to Chrome  I see all icons again well.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @magicse ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
